widgets/GCSBucketCreate-action.json

```diff
@@ -2,7 +2,7 @@
   "metadata": {
     "spec-version": "1.5"
   },
-  "display-name" : "Google Cloud Storage Create",
+  "display-name" : "GCS Create Bucket",
```

Comment:
This plugin can create more than just buckets, it can also create objects within a bucket (The class is named poorly). Should we just call it "GCS Create"?

Reply:
done